haskell.md

```diff
@@ -320,6 +320,33 @@ Imports *should* be grouped in the following order:
 
 Put a blank line between each group of imports.
 
+For qualified imports you *should* use `ImportQualifiedPost` [extension](https://ghc.gitlab.haskell.org/ghc/doc/users_guide/exts/import_qualified_post.html).
+In case project uses GHC < 8.10.1 you *should* put `qualified` imports in
```

Comment:
I would say "supports" instead of "uses" and replace "project" with "package". Libraries often support multiple GHC versions and if at least one of them is <8.10, we shouldn't use postfix form.

Reply:
I've made these changes.